example/app/views/chart.html

```diff
@@ -10,7 +10,7 @@
     <script src="/assets/d3.min.js"></script>
     <script src="/chart.js"></script>
     <script>
-    var chart = characterChart(d3.select("#chart"), "/csv/{{slug}}.csv");
+    var chart = characterChart(d3.select("#chart"), "/csv/{{slug}}.csv", "/csv/episodes.csv");
```

Comment:
The function receives a path like `/very/custom/path/Donald_Trump.csv`. Then you remove the filename and ` `/very/custom/path/episodes.csv` contains the episodes, ` `/very/custom/path/2016-03.csv` the data for this month etc.